qa/suites/upgrade/luminous-x/parallel/0-cluster/start.yaml

```diff
@@ -0,0 +1,34 @@
+meta:
+- desc: |
+   Run ceph on two nodes,
+   with a separate client 0,1,2 third node.
+   Use xfs beneath the osds.
+   CephFS tests running on client 2,3
+roles:
+- - mon.a
+  - mgr.x
+  - mds.a
+  - osd.0
+  - osd.1
+- - mon.b
+  - mon.c
+  - osd.2
+  - osd.3
+- - client.0
+  - client.1
+  - client.2
+  - client.3
+overrides:
+  ceph:
+    log-whitelist:
+    - scrub mismatch
+    - ScrubResult
+    - wrongly marked
+    - (POOL_APP_NOT_ENABLED)
+    - overall HEALTH_
+    conf:
+      global:
+        enable experimental unrecoverable data corrupting features: "*"
```

Comment:
@yuriw I feel should be `corruption` instead of `corrupting`.
What your thoughts?

Reply:
@amitkumar50 I actually don't know :(